linode/manager
# Repo Clean Up 🧹 🧼

## Description

- Removes some old stuff we don't use and have no intention to bring back any time soon with intentions to keep the repo clean and focused

## How to test

- Check all components such as unit tests, e2e tests, Storybook, and Cloud Manager
- Ensure I didn't remove anything you care about
